swri_route_util/src/route_speeds.cpp

```diff
@@ -355,10 +355,27 @@ void speedsForObstacles(
       }
       continue;
     }
+    
+    // Use half of the vehicle width override as the car radius when smaller
+    // This is used for tight routes that are known to be safe such as when going
+    // though cones. It prevents objects close to the vehicle from causing the vehicle
+    // to stop/slow down too much unnecessarily.
+    double veh_r = car_r;
+    if (point.hasProperty("vehicle_width_override"))
+    {
+      ROS_DEBUG("Speeds for obstacle found vehicle_width_override property");
+      double width = point.getTypedProperty<double>("vehicle_width_override");
+
+      // Pick the smaller of the radii
+      if (veh_r >= width/2.0)
+      {
+        veh_r = width/2.0;
```

Comment:
It probably makes sense to issue a warning here if the vehicle width is actually overridden. Doing so won't change the behavior, but it will make it more obvious if it causes problems. Given Marc's earlier comment about spamming the console, a `ROS_WARN_THROTTLE(1.0, ...)` is probably appropriate.

Reply:
Good point.

Reply:
Added